popup-style.css

```diff
@@ -0,0 +1,476 @@
+    font-family: 'Poppins', sans-serif;
+  }
+
+  .fontimg1popup1 {
+    display: none;
+  }
+
+  .fontimg2popup1 {
+    height: 55vh;
+  }
+
+  .factorypopup1 {
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 18px;
+    color: #344563;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .roundpoppup1 {
+    background-color: darkgray;
+    height: 10px;
+    width: 10px;
+    margin-top: 5px;
+    border-radius: 50%;
+    margin-left: 10px;
+    margin-right: 10px;
+  }
+
+  .see-projectpopup1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    gap: 10px;
+    width: 152px;
+    height: 35px;
+    border-radius: 8px;
+    color: #396df2;
+    font-size: 17px;
+    line-height: 24px;
+    border: 1px solid;
+    border-color: #162ae3;
+    background-color: hsl(0, 0%, 100%);
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+  }
+
+  .see-projectpopup1 a {
+    text-decoration: none;
+    color: #396df2;
+    background-color: hsl(0, 0%, 100%);
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+  }
+
+  .see-projectpopup11 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    padding: 10px;
+    gap: 10px;
+    width: 152px;
+    height: 35px;
+    border-radius: 8px;
+    color: #396df2;
+    font-size: 17px;
+    line-height: 24px;
+    border: 1px solid;
+    border-color: #162ae3;
+    background-color: hsl(0, 0%, 100%);
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+  }
+
+  .see-projectpopup11 a {
+    text-decoration: none;
+    color: #396df2;
+    background-color: hsl(0, 0%, 100%);
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+  }
+
+  .languagespopup1 {
+    position: relative;
+    display: flex;
+  }
+
+  .languagespopup1 li {
+    list-style: none;
+  }
+
+  .flexpop-bnt {
+    display: flex;
+    justify-content: space-between;
+    justify-self: start;
+  }
+
+  .flex1popup1 {
+    display: flex;
+    margin: 0;
+    padding: 0;
+  }
+
+  .flex-desk1 {
+    display: flex;
+    display: block;
+    flex-direction: column;
+    height: 40px;
+    width: 80%;
+  }
+
+  .tonic1popup1 {
+    display: flex;
+    min-height: 10vh;
+  }
+
+  .tonic1popup1-text {
+    position: relative;
+    width: 80%;
+    padding: 0;
+  }
+
+  .languagepopup1 {
+    display: inline;
+    padding: 4px 12px;
+    border-radius: 8px;
+    background-color: #ebebff;
+    font-size: 15px;
+    color: #0069d9;
+    margin-right: 20px;
+    margin-bottom: 10px;
+  }
+
+  .languagespopup11 {
+    position: relative;
+    display: flex;
+  }
+
+  .popup1 {
+    display: none;
+    z-index: 9999;
+    top: 5px;
+    background-color: white;
+    left: 2%;
+    border: 1px solid #c1c7d0;
+    border-radius: 16px;
+    position: fixed;
+    bottom: 5px;
+    width: 95%;
+    height: auto;
+  }
+
```

Comment:
- [ ] It would be nice if you could improve on the styles of the popup modal so it matches the Figma design given. Currently, there is no padding within the popup and everything touches the edges of the popup. Kindly add some padding and then align items nicely with breathing spaces just as I marked them on the screenshots below. 👍🏽
- [ ] Kindly implement the changes and improvements on both the mobile and desktop screens for all the popups on the page. 👍🏽

**The Original Designs**
<img width="874" alt="Screenshot 2023-03-16 at 1 51 46 PM" src="https://user-images.githubusercontent.com/50941074/225642198-ba1974b4-679d-43cc-94ea-95293295a6d4.png">

<img width="377" alt="Screenshot 2023-03-16 at 1 47 45 PM" src="https://user-images.githubusercontent.com/50941074/225642724-c2ffea66-9dc7-4e39-a2e9-b0d73cdc5490.png">


**Your Designs**
<img width="1761" alt="Screenshot 2023-03-16 at 1 53 32 PM" src="https://user-images.githubusercontent.com/50941074/225642391-055f8c21-f18a-442f-a211-91e1f1389170.png">

<img width="395" alt="Screenshot 2023-03-16 at 1 49 17 PM" src="https://user-images.githubusercontent.com/50941074/225643005-c395be64-bd0e-4903-a7c0-c52096894eb9.png">